suite/regress/arm_v5.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+
+# Test ARMv5 mode assembles only supported features.
+
+# Author: Andrew O'Brien
+
+from keystone import (Ks, KsError, KS_ARCH_ARM, KS_MODE_ARM, KS_MODE_V5)
+import regress
+
+class TestARM(regress.RegressTest):
+    asm = b"""
+    MOVT r0, #0
+    """
```

Comment:
this sample code can also assembled by armv7 i think, with the same result. is there any code that is only available in armv5?